<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Mazes</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="mazeStyleTab">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>451</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="style1Tab">
     <attribute name="title">
      <string>Style1</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="style2Tab">
     <attribute name="title">
      <string>Style2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="style3Tab">
     <attribute name="title">
      <string>Style3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="style4Tab">
     <attribute name="title">
      <string>Style4</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="style5Tab">
     <attribute name="title">
      <string>Style5</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="manualTab">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QSpinBox" name="widthSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>70</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Width:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Height:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="heightSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>130</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="typeComboBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="generateButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>450</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>239</x>
      <y>440</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="isometricButton">
       <property name="text">
        <string>Isometric</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="solveButton">
       <property name="text">
        <string>Solve</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="numberOfMazeSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>195</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>195</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Of Maze:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="loopCheckBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>With Loop:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="wallColorButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Wall Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floorColorButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Floor Color</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskFileDialogButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Mask File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="maskFileEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>201</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="isometricBackgroundButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>130</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Isometric Background</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>719</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAssetLocation"/>
    <addaction name="actionSaveLocation"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAssetLocation">
   <property name="text">
    <string>Asset Location</string>
   </property>
  </action>
  <action name="actionSaveLocation">
   <property name="text">
    <string>Save Location</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
